data/test/macros/wml_unit_test_macros.cfg

```diff
@@ -71,7 +71,7 @@ human#endarg
 Elvish Archer#endarg
 
     [test]
-        name = "Unit Test " + {NAME}
+        name = _ "Unit Test " + {NAME}
```

Comment:
This doesn't feel like a good change - if the schema is checking for a mark, but isn't complaining about the untranslatable `NAME`.

Reply:
A value is considered translatable if any segment of it is translatable, and it's fairly common to concatenate variously-translatable parts in this way (for example, untranslatable formatting codes and translatable content). This at least avoids the trap of putting a macro inside the translatable string, and since this is in wesnoth-test, it won't be sent to the translators anyway.